magma/bind.py

```diff
@@ -83,30 +83,32 @@ def _bind(cls, monitor, compile_fn, *args):
         bind_arg = temp
         ports += _gen_bind_port(cls, mon_arg, bind_arg)
     ports_str = ",\n    ".join(ports)
-    bind_str = f"bind {cls.name} {monitor.name} {monitor.name}_inst (\n    {ports_str}\n);"  # noqa
+    cls_name = cls.name
```

Comment:
is there any hope of not special casing this logic here and resolving the verilog name using a "symbol table"?

Reply:
I think in this case the better long term solution is to just push the bind logic down into coreir, so the name resolution occurs there.  This is just a hot fix since FB is using user_namespace now and that broke the verification flow using bind.